fe/fe-core/src/test/java/org/apache/doris/utframe/TestWithFeService.java

```diff
@@ -540,4 +547,19 @@ private void checkAlterJob() throws InterruptedException, MetaNotFoundException
         }
     }
 
+    // clear the specified dir
```

Comment:
Why not use FileUtils.deleteDirectory

Reply:
> Why not use FileUtils.deleteDirectory

If so, Exception, [unable-to-delete-directory](https://stackoverflow.com/questions/50532263/java-delete-directory-ioexception-unable-to-delete-directory) will be thrown.

So i reference [createAndClearDir](https://github.com/apache/doris/blob/0dd2fb758cc777284b4506d8ceaa51fc4603a5cf/fe/fe-core/src/test/java/org/apache/doris/utframe/MockedFrontend.java#L151)